README.md

```diff
@@ -10,9 +10,9 @@ Azure Data Studio is a data management tool that enables you to work with SQL Se
 
 |Platform |Type             |Download |
 | --------|-----------------|-------- |
-|Windows  |User Installer   |[64 bit][win-user] |
-|         |System Installer |[64 bit][win-system] |
-|         |.zip             |[64 bit][win-zip] |
+|Windows  |User Installer   |[64 bit][win-user]&emsp;[ARM][in-win-user-arm64] |
+|         |System Installer |[64 bit][win-system]&emsp;[ARM][in-win-system-arm64] |
+|         |.zip             |[64 bit][win-zip]&emsp;[ARM][in-win-zip-arm64] |
```

Comment:
Are the in- supposed to be there?

```suggestion
|Windows  |User Installer   |[64 bit][win-user]&emsp;[ARM][win-user-arm64] |
|         |System Installer |[64 bit][win-system]&emsp;[ARM][win-system-arm64] |
|         |.zip             |[64 bit][win-zip]&emsp;[ARM][win-zip-arm64] |
```

Reply:
good catch, I copied them from insider builds table and missed updating them.